### ISSUE TITLE
Pre download instead of stream 

### ISSUE THREAD

**Issue by ishampadron** [popcorn-official/popcorn-android#812]
Would like to be able to see options for download instead of stream.

**Comment by virajkarandikar** [comment]
Yes. Please add a download option, so that media can be downloaded/moved and watched later (without network).

**Comment by mickey66092** [comment]
Yea indeed, in the rlly old release years back it was intergrated so why removing it. Rlly annoying because you cant preload anymore and watch it on the go or later.

**Comment by KoustuvGanguly** [comment]
Yes, badly missing this feature. Please bring back download option. Also please provide resolution selection in movie details page.

**Comment by ithakaa** [comment]
+1 for the feature reuest
